istioctl/pkg/writer/envoy/configdump/configdump.go

```diff
@@ -60,6 +60,22 @@ func (c *ConfigWriter) PrintBootstrapDump() error {
 	return nil
 }
 
+// PrintEndpointDump prints just the endpoint config dump to the ConfigWriter stdout
+func (c *ConfigWriter) PrintEndpointDump() error {
```

Comment:
is this used anywhere?

Reply:
I added this for testing, I will remove it.